src/loader/bgw_launcher.c

```diff
@@ -105,12 +94,21 @@ report_bgw_limit_exceeded(void)
 static void
 report_error_on_worker_register_failure(void)
 {
-	ereport(ERROR, (errcode(ERRCODE_INSUFFICIENT_RESOURCES),
-					errmsg("No available background worker slots."),
-					errhint("TimescaleDB background worker launcher shutting down. It will be restarted by the postmaster.")));
+	ereport(LOG, (errcode(ERRCODE_INSUFFICIENT_RESOURCES),
```

Comment:
I think we need an comment on why this isn't a hard error here. When background workers actually do something will we want to up this to `WARNING`?

Reply:
LOG is higher than warning.